app/models/filterable_search.rb

```diff
@@ -47,6 +49,10 @@ def initialize_date_attributes(options)
     end
   end
 
+  def initialize_facet_attributes(options)
```

Comment:
Is the plan to expand this method in future PRs to support other facet attributes?

Reply:
Yep, see: https://github.com/GSA/search-gov/pull/1071/files#diff-862beb065f9ac1a20d2b969361d71e64770fd956524da00c39c952cf510e4688 in the other open PR.